src/Components/App/App.js

```diff
@@ -61,22 +61,24 @@ class App extends Component {
   }
 
   addTrack(track) {
-    const playlistTracks = this.state;
+    const { playlistTracks } = this.state;
```

Comment:
You gotta "destructure" this out of state. The way you had it `playlistTracks` was the whole `state` object.